src/main/pages/setup-docker/docker-config.adoc

```diff
@@ -383,12 +406,13 @@ docker run -it --rm -v /var/run/docker.sock:/var/run/docker.sock \
                        eclipse/che:<version> [COMMAND] --debug
 ----
 
-You can also optionally use your local binaries in production mode by volume mounting `:/repo` without passing `--debug`. There are two locations that files in your Che source repository will be used instead of those in the image:
+Optionally, you can use your local binaries in production mode by volume mounting `:/repo` without passing the `--debug` argument. Files from two locations from your Che source repository will be used instead of those in the image:
 
-1.  During the `che config` phase, the source repository’s `/dockerfiles/init/modules` and `/dockerfiles/init/manifests` will be used instead of the ones that are included in the `eclipse/che-init` container.
-2.  During the `che start` phase, a local assembly from `assembly/assembly-main/target/` is mounted into the `eclipse/che-server` runtime container. You must `mvn clean install` the `assembly/assembly-main/` folder prior to activating development mode.
+* During the `che config` phase, the source repository’s `/dockerfiles/init/modules` and `/dockerfiles/init/manifests` will be used instead of the ones that are included in the `eclipse/che-init` container.
 
-Volume mounting `:/repo` will also make use of your repository’s puppet manifests and other files (replacing those that are stored within the CLI’s base image). If you only want to volume mount a new set of assemblies and ignore the other items in a repository, you can do so by volume mounting `:/assembly` to a folder that is the base of a binary (we do not yet support volume mounting a `.tgz` file).
+* During the `che start` phase, a local assembly from `assembly/assembly-main/target/` is mounted into the `eclipse/che-server` runtime container. You must `mvn clean install` the `assembly/assembly-main/` folder prior to activating development mode.
+
+Volume mounting `:/repo` will also make use of your repository’s puppet manifests and other files (replacing those that are stored within the CLI’s base image). To only volume mount a new set of assemblies and ignore the other items in a repository, volume mount `:/assembly` to a folder that is the base of a binary. Volume mounting a `.tgz` file is not supported as yet.
```

Comment:
* volume mounting -- 4 times
* 'CLI's base image' -- just name the image; 'CLI' isn't a good specifier